src/components/feedback-particle-effect/feedback-text-effects.ts

```diff
@@ -1,159 +1,85 @@
-      "message",
-      this.handleTextWorkerMessage.bind(this)
-    );
+  constructor() {
+    this.feedbackTextElement = document.getElementById("feedback-text");
+    this.initialize();
   }
 
-  public wrapText(text: string): void {
-    const gradient = this.context.createLinearGradient(
-      0,
-      0,
-      this.canvasWidth,
-      this.canvasHeight
-    );
-    gradient.addColorStop(0.3, "#F8E218");
-    gradient.addColorStop(0.5, "#F8E218");
-    gradient.addColorStop(0.7, "#E39D37");
-    this.context.fillStyle = gradient;
-    this.context.textAlign = "center";
-    this.context.textBaseline = "middle";
-    this.context.lineWidth = 3;
-    this.context.strokeStyle = "#A46225";
-    this.context.font = `${
-      this.fontSize - text.length * 0.3
-    }px ${font}, monospace`;
-    // break multiline text
-    let lineArray: string[] = [];
-    let words = text.split(" ");
-    let lineCounter = 0;
-    let line = "";
-    for (let i = 0; i < words.length; i++) {
-      let testLine = line + words[i] + " ";
-      if (this.context.measureText(testLine).width > this.maxTextWidth) {
-        line = words[i] + "";
-        lineCounter++;
-      } else {
-        line = testLine;
-      }
-      lineArray[lineCounter] = line;
-    }
-    let textHeight = this.lineHeight * lineCounter;
-    this.textY = this.canvasHeight / 4.2 - textHeight / 2;
-    text = text.trim();
-    lineArray.forEach((text, index) => {
-      let lastSpaceIndex = text.lastIndexOf(" ", text.lastIndexOf(" ") - 1);
-      if (
-        this.fontSize * text.length > this.canvasWidth * 1.7 &&
-        lastSpaceIndex != -1
-      ) {
-        let initialText = text.slice(0, lastSpaceIndex);
-        let lastText = " " + text.slice(lastSpaceIndex + 1);
-        this.context.fillText(
-          initialText,
-          this.textX,
-          this.textY + index * this.lineHeight
-        );
-        this.context.fillText(
-          lastText,
-          this.textX,
-          this.textY + index * this.lineHeight + this.canvasHeight / 12
-        );
-      } else {
-        this.context.fillText(
-          text,
-          this.textX,
-          this.textY + index * this.lineHeight
-        );
-      }
-    });
-    this.convertToParticle();
+  /**
+   * Initializes the feedback text effects, loading the font if the element is found.
+   */
+  private initialize() {
+    if (!this.isFeedbackElementAvailable()) return;
+    this.loadFont();
   }
 
-  private handleTextWorkerMessage(event: MessageEvent): void {
-    this.particles = event.data.map(
-      ({ x, y, color }) => new TextParticle(this, x, y, color)
-    );
+  /**
+   * Checks if the feedback text element is available and logs an error if not.
+   * @returns A boolean indicating whether the element is available.
+   */
+  private isFeedbackElementAvailable(): boolean {
+    if (!this.feedbackTextElement) {
+      console.error("Feedback text element not found!");
+      return false;
+    }
+    return true;
```

Comment:
Question, why don't we just return the` return !this.feedbackTextElement` ?
Do we really need that console.error that badly? This will not even be seen in the production.

Reply:
applied this @bernhardccodev thank you